### PR TITLE
Add unit tests rule to Makefile.

### PR DESCRIPTION
**Pull request by kevinrizza**

### CODE REVIEW

**Comment by kevinrizza** [comment]
/cc @aravindhp 

Looks like it didn't add you by default.

File: Makefile [hunk 1]
```diff
@@ -20,6 +20,9 @@ osbs-build:
 	# hack/build.sh
 	./tmp/build/build.sh
 
+unit:
+	go test -v ./pkg/...
```
Comment (aravindhp) [inline Makefile]: Is this what other teams are doing? We should ensure that this doesn't cause trouble when launched by the CI framework. Is there a way to do that?

Comment (kevinrizza) [reply Makefile]: It's being used exactly this way by the OLM team. I actually just copied their pattern for this bit:

https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Makefile#L21

https://github.com/openshift/release/blob/master/ci-operator/config/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master.yaml#L38

**Comment by aravindhp** [comment]
@tkashem please review

File: Makefile (hunk 1 above)
Comment (tkashem) [inline Makefile]: @kevinrizza My bad, I forgot to mention that we do not commit the generated mock files into git. So we would need to generate the mock files before the unit tests can be run.  `make generate-mocks` will generate the mocks for you in the right place.

Comment (aravindhp) [reply Makefile]: Please open a Jira bug to track this.

**Pull request merged**